generator/uscore/uscore_generator.rb

```diff
@@ -659,22 +688,32 @@ def fixed_value_search?(search_parameters, sequence)
 
       def get_first_search_by_patient(sequence, search_parameters, save_resource_ids_in_bundle_arguments)
         %(
-          #{get_search_params(search_parameters, sequence)}
-          reply = get_resource_by_params(versioned_resource_class('#{sequence[:resource]}'), search_params)
-          assert_response_ok(reply)
-          assert_bundle_response(reply)
+          @#{sequence[:resource].underscore}_ary = {}
```

Comment:
Since this is now a hash, I'd love it if it wasn't named `*_ary`. The US Core resources work out so that `*s` works as a plural for them all.

Reply:
I was wondering if this would make more sense, and quickly scanned to see if we could just replace with 's'.  But I didn't end up doing it because it is easy to drop an `s` in places by mistake in the generator.  But I can readd.